tests/unit/memefficiency.tcl

```diff
@@ -37,7 +37,6 @@ start_server {tags {"memefficiency"}} {
 }
 
 start_server {tags {"defrag"}} {
-    if {[string match {*jemalloc*} [s mem_allocator]]} {
         test "Active defrag" {
```

Comment:
Please fix indentation, everything else looks good :)

Reply:
Done -  ci/jenkins/redis and ci/jenkins/redis_memkind configuration job should be updated @kasiawasiuta